app/models/competition.rb

```diff
@@ -950,6 +950,10 @@ def can_edit_registration_fees?
   end
 
   def registration_opened?
+    use_wca_registration? && !cancelled? && !registration_not_yet_opened?
+  end
+
+  def registration_is_open?
```

Comment:
As per Ruby conventions, boolean-esque methods don't incude any modal verbs like `is`, `has` or `should`. The question mark itself is already the "is" part.